File: Angular/my-angular-app/src/app/_models/Report.ts
export class Report {
    constructor(
        public id: number = 0,
        public location: string = "",
        public img: string = "",
        public date: Date = new Date(),
        public userId: string = "",
        public speciesId: number = 0,
        public speciesName: string=""
    ) {}
}

File: API/scenario10API/scenario10API/models/Report.cs
namespace scenario10API.models
{
    public class Report
    {
        public int Id { get; set; }
        public string Location { get; set; } = "";
        public string Img { get; set; } = "";
        public DateTime Date { get; set; }
        public string UserId { get; set; } = "";
        public int SpeciesId { get; set; }
        public Species? Species { get; set; }
        public User? User { get; set; }

    }
}
